configs/docker.rc

```diff
@@ -79,3 +79,5 @@ docker_certbot =
 
 # Should we return a restrictive robots.txt?
 disallow_robots =
+
+docker_elasticsearch_version = 5.6.2
```

Comment:
It this possible to move this to `base_env.rc` so we can use it for both docker and fabric with `env.docker_elasticsearch_version` instead of the constant `ELASTICSEARCH_VERSION` in `fabfile.py`? This way we only have one place to change the version.